cachecontrol/caches/redis_cache.py

```diff
@@ -20,7 +20,13 @@ def __init__(self, conn):
         self.conn = conn
 
     def get(self, key):
-        return self.conn.get(key)
+        # a key might be of the format foo;bar (where `;` is our default delimter).
+        # To get this key's value from redis we do `mget foo foo;bar` (this
+        # returns a list of values for each key searched for) and return the
+        # first match
+        key_prefix, _ = keybreaker(key)
+        key = [key_prefix, key]
+        return next((i for i in self.conn.mget(key) if i), None)
```

Comment:
I think `next` is not required here, `mget` will return None if key doesn't exist so this can be simplified to
```python
values = self.conn.mget([key_prefix, key])
return values[0] or values[1]
```